sdks/go/pkg/beam/register/register_test.go

```diff
@@ -672,3 +675,133 @@ func (fn *PartialCombiner2) AddInput(i int, c CustomType) int {
 func (fn *PartialCombiner2) MergeAccumulators(i1 int, i2 int) int {
 	return i1 + i2
 }
+
+// Foo is a struct with a method for measuring method invocation
+// overhead for StructuralDoFns.
+type Foo struct {
+	A int
+}
+
+// ProcessElement is a method for measuring a baseline of structural dofn overhead.
+func (f *Foo) ProcessElement(b CustomType) int {
+	return f.A + b.val
+}
+
+func MakeMultiEdge(f *graph.DoFn) graph.MultiEdge {
+	return graph.MultiEdge{
+		DoFn: f,
+	}
+}
+
+type callerCustomTypeГint struct {
```

Comment:
```suggestion
type callerCustomTypeГint struct {
```
is this character intentional?

Reply:
okay.. interesting!